src/basic/openssl-util.c

```diff
@@ -46,4 +47,42 @@ int hmac(
 
         return 0;
 }
+
+int openssl_hash(
+        const EVP_MD *alg,
+        const void *msg,
+        size_t msg_len,
+        uint8_t *md,
+        unsigned int *md_len) {
+        _cleanup_(EVP_MD_CTX_freep) EVP_MD_CTX *ctx = NULL;
+        unsigned int len;
```

Comment:
size_t

Reply:
this one will conflict with openssl function signatures, but it'll be wrapped by size_t